client/cli/src/lib.rs

```diff
@@ -119,10 +127,13 @@ pub trait SubstrateCli: Sized {
 		let mut full_version = Self::impl_version().to_string();
 		full_version.push_str("\n");
 
+		let name = Self::executable_name();
+		let author = Self::author();
+		let about = Self::description();
 		let app = app
-			.name(Self::executable_name())
-			.author(Self::author())
-			.about(Self::description())
+			.name(&*name)
+			.author(&*author)
+			.about(&*about)
```

Comment:
```suggestion
			.name(name)
			.author(author.as_str())
			.about(about.as_str())
```
I usually prefer to use `.as_str()` as it is more descriptive of what it does. But it's just a taste, do what you prefer

Reply:
https://github.com/paritytech/substrate/pull/6550/commits/66a8c331a69f4e2f226ce0328aabe73cabc57451